.github/workflows/django.yml

```
@@ -0,0 +1,36 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+defaults:
+  run:
+    working-directory: ./pprxweb
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Run Tests
+      env:
+        DJANGO_SECRET_KEY: "test"
+        CLIENT_ID: "test"
+        CLIENT_SECRET: "test"
+        GOOGLE_USERNAME: "test"
+        GOOGLE_APP_PASSWORD: "test"
+      run: |
```

```suggestion
      env:
        DJANGO_SECRET_KEY: ${{ secrets.DJANGO_SECRET_KEY }}
        CLIENT_ID: ${{ secrets.CLIENT_ID }}
        CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
        GOOGLE_USERNAME: ${{ secrets.GOOGLE_USERNAME }}
        GOOGLE_APP_PASSWORD: ${{ secrets.GOOGLE_APP_PASSWORD }}
      run: |
```
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions